File: java/src/main/java/P2011_Performing_Operations.java
public class P2011_Performing_Operations {

    public static void main(String args[]) {

        String[] array = { "--X", "X++", "X++" };
        System.out.println(new P2011_Performing_Operations().finalValueAfterOperations(array));

    }

    public int finalValueAfterOperations(String[] operations) {

        int value = 0;

        for(String s: operations){
            if(s.contains("--"))
                value -= 1;
            else if(s.contains("++"))
                value += 1;
        }

        return value;
    }

}

File: python/P2011_Performing_Operations.py
from typing import List

class Solution:
    def finalValueAfterOperations(self, operations: List[str]) -> int:
        value = 0
        
        for x in operations:
            if(x == "++X" or x == "X++"):
                value += 1
            elif(x == "--X" or x == "X--"):
                value -= 1
        
        return value
        

solution = Solution()
array = ["--X","X++","X++"]
finalValue = solution.finalValueAfterOperations(array)
print(finalValue)
